README.md

```diff
@@ -62,7 +67,7 @@ This is only necessary when you want to build the extension yourself from source
 	$ yarn run run:firefox
 	~~~~
 
-	When using Yarn, just replace `npm` with `yarn`.
+	When using Npm, just replace `yarn` with `npm`.
```

Comment:
Same

Reply:
should I remove the npm reference, then ?

Reply:
Yes